pkg/handlers/forwarder.go

```diff
@@ -28,6 +29,15 @@ import (
 
 const defaultFlushInterval = time.Duration(100) * time.Millisecond
 
+// forwarderCustomErrHandler is originally implemented to avoid having
+// the following log `http: proxy error: context canceled` printed by Golang
+func forwarderCustomErrHandler(w http.ResponseWriter, r *http.Request, err error) {
+	if err != context.Canceled {
+		log.Println("forwarder:", err)
```

Comment:
Perhaps like you need to add logger.LogIf for consistency..

Reply:
> Perhaps like you need to add logger.LogIf for consistency..

vet forbids it (although I don't see any circular dependency)

```
11:04 $ make vet
Running vet
go: finding github.com/eapache/go-xerial-snappy latest
import cycle not allowed
package github.com/minio/minio
	imports github.com/minio/minio/cmd
	imports github.com/minio/minio/cmd/crypto
	imports github.com/minio/minio/cmd/logger
	imports github.com/minio/minio/cmd/logger/message/audit
	imports github.com/minio/minio/pkg/handlers
	imports github.com/minio/minio/cmd/logger
can't load package: import cycle not allowed
package github.com/minio/minio
	imports github.com/minio/minio/cmd
	imports github.com/minio/minio/cmd/crypto
	imports github.com/minio/minio/cmd/logger
	imports github.com/minio/minio/cmd/logger/message/audit
	imports github.com/minio/minio/pkg/handlers
	imports github.com/minio/minio/cmd/logger
Makefile:32: recipe for target 'vet' failed
make: *** [vet] Error 1
```

Reply:
oh it's not possible in that manner. You need to take the logger.LogIf from initializing NewForwarder()